rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/schedule/ScheduleAdminController.java

```diff
@@ -30,33 +31,81 @@
 import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
-
+    
 @BasePathAwareController
 @RequiredArgsConstructor(onConstructor = @__(@Autowired))
 @RestController
 @SecurityRequirement(name = "tokenAuth")
 @SecurityRequirement(name = "basic")
 public class ScheduleAdminController implements RepresentationModelProcessor<RepositoryLinksResource> {
     public static final String SCHEDULE_URL = "/schedule";
-    
+
     @NonNull
     private final RestControllerHelper restControllerHelper;
-    
+
     @NonNull
     private Sw360ScheduleService scheduleService;
-    
+
 
     @Override
     public RepositoryLinksResource process(RepositoryLinksResource resource) {
         resource.add(linkTo(ScheduleAdminController.class).slash("api/schedule").withRel("schedule"));
         return resource;
     }
 
-    @RequestMapping(value = SCHEDULE_URL + "/unscheduleAllServices", method = RequestMethod.DELETE)
+    @RequestMapping(value = SCHEDULE_URL + "/unscheduleAllServices", method = RequestMethod.POST)
     public ResponseEntity<?> unscheduleAllServices()throws TException {
         User sw360User = restControllerHelper.getSw360UserFromAuthentication();
         RequestStatus requestStatus = scheduleService.cancelAllServices(sw360User);
         HttpStatus status = HttpStatus.OK;
         return new ResponseEntity<>(requestStatus, status);
     }
+    
+    @RequestMapping(value = SCHEDULE_URL + "/cveService", method = RequestMethod.POST)
+    public ResponseEntity<?> scheduleCve()throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        RequestSummary requestSummary = scheduleService.scheduleCveSearch(sw360User);
+        HttpStatus status = HttpStatus.OK;
```

Comment:
Please consider using `202 ACCEPTED` instead of `200 OK` as no immediate action is taken for all endpoints as done in #2269.